participants/models.py

```diff
@@ -24,6 +25,10 @@ def count_voted(self):
     def count_total_participants(self):
         return Participant.objects.filter(contest__year=self.contest.year).count()
 
+    @property
+    def cached_total_score(self):
+        return cache.get(f'participant{self.id}')
```

Comment:
2 points:
1. it's wise to keep key name template as a setting/constant, and then use it in both get and set (and everywhere else)
1. key template prefix/suffix is usually separated like `participant:{self.id}` or `participant.{self.id}`